src/components/discovery/DiscoverySearchForm.js

```diff
@@ -92,9 +55,17 @@ PhenopacketDropdownOption.propTypes = {
   getDataTypeFieldSchema: PropTypes.func,
 };
 
-const DiscoverySearchForm = ({ onChange, dataType, setFormRef, handleVariantHiddenFieldChange }) => {
+const DiscoverySearchForm = ({
+  onChange,
+  dataType,
+  setFormRef,
+  handleVariantHiddenFieldChange,
+  conditionValidator,
+}) => {
   const [form] = Form.useForm();
 
+  const CONDITION_RULES = [{ validator: conditionValidator }];
```

Comment:
since this is inside a function, it shouldn't be all-caps constant-named anymore - just a normal variable

Reply:
or, if conditionValidator is treated as an import instead, this could stay a constant outside of the component function.

Reply:
@gsfk can you move this back outside of DiscoverySearchForm to avoid redefining it every render?

Reply:
ah, managed to overlook this while moving everything around